### ISSUE TITLE
Support other alphabets in notebook names

### ISSUE THREAD

**Issue by ronja-ui** [teragrep/ajs_01#305]
**Description**
At the moment Teragrep supports only latin alphabets (and couple symbols) in notebook names. However, other alphabets (e.g. Ä, Ö, Å for starters) should be supported to increase the language accessibility.
